setup.py

```diff
@@ -2,9 +2,13 @@
     import regex as re
 except ImportError:
     import re
-from setuptools import find_packages, setup
+from setuptools import setup  # , find_packages  # unused
```

Comment:
Why comment it out and not remove it? Why add `# unused`?

Reply:
Because if it turns out that we need it, we can always uncomment it. You can remove it if you want